### PR TITLE
rel to #7911: change to new cgeo specific fulldata API

### PR DESCRIPTION
**Pull request by eddiemuc**
rel to #7911: change to new cgeo specific fulldata API

### CODE REVIEW

**Comment by eddiemuc** [comment]
Note: this PR should be merged rather quickly. Until then bettercacher.org integration uses the "full API" to gather missing caches on map search (including e.g. logs).

File: main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java [hunk 1]
```diff
@@ -292,9 +237,9 @@ public Geocache createGeocache() {
             cache.setCoords(GeopointParser.parse(lat + " " + lng, null));
             // "type": "unknown",
             cache.setType(getFromJson(CACHE_TYPE_MAP, type, CacheType.UNKNOWN));
-            // "difficulty": "4_5",
+            // "difficulty": "4.5",
```
Comment (Lineflyer) [inline main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java:240]: Just my few cents:
Is this a left over or for documentation purpose. For the latter it should probably be explained, what it shall tell.

File: main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java [hunk 2]
```diff
@@ -292,9 +237,9 @@ public Geocache createGeocache() {
...
             cache.setDifficulty(getNumberFromJson(difficulty, 0));
-            // "terrain": "1_5",
+            // "terrain": "1.5",
```
Comment (Lineflyer) [inline main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java:242]: see above

File: main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java (hunk 1 above)
Comment (Lineflyer) [reply main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java:240]: Ah OK :)

**Pull request merged**